README.md

```diff
@@ -87,7 +87,7 @@ Exchanges may be centralized (**CEX**), or decentralized (**DEX**), in which cas
 | ![](https://img.shields.io/static/v1?label=Hummingbot&message=BRONZE&color=green) | [Sushiswap](https://docs.hummingbot.org/exchanges/sushiswap/) | AMM DEX |
 | ![](https://img.shields.io/static/v1?label=Hummingbot&message=BRONZE&color=green) | [Tinyman](https://docs.hummingbot.org/exchanges/tinyman/) | SPOT DEX |
 | ![](https://img.shields.io/static/v1?label=Hummingbot&message=BRONZE&color=green) | [Traderjoe](https://docs.hummingbot.org/exchanges/traderjoe) | AMM 
-| ![](https://img.shields.io/static/v1?label=Hummingbot&message=BRONZE&color=green) | [Uniswap](https://docs.hummingbot.org/exchanges/uniswap/) | AMM DEX |
+| ![](https://img.shields.io/static/v1?label=Hummingbot&message=BRONZE&color=green) | [Amm](https://docs.hummingbot.org/exchanges/amm/) | AMM DEX |
```

Comment:
That's exchange, not strategy

Reply:
So i have to change this ?

Reply:
@nikspz

Reply:
We have connector / exchange named uniswap
https://hummingbot.org/exchanges/uniswap/

this changes here not needed

Reply:
ok
any other changes ? @nikspz  you want

Reply:
@nikspz  done with all changes you said plz check now